src/main/java/org/jenkinsci/account/Application.java

```diff
@@ -224,6 +224,21 @@ private boolean badNameElement(String userid) {
         return Pattern.matches("^[sdfghrt0-9]+$", userid.toLowerCase());
     }
 
+    public String geoIp(String ip) {
+
+        try {
+            URL url = new URL("http://freegeoip.net/csv/" + ip);
```

Comment:
They do, but I was testing it locally, and getting a SSL cert issue... I'll take a look again.